### ISSUE TITLE
Player stopped working

### ISSUE THREAD

**Issue by Windyjunk** [Heckie75/kodi-addon-bbc-podcasts#5]
I have this addon installed on 4 kodi builds, two on Android TV OS, one on Google TV OS, and one on windows 11. All have stopped functioning, there is no content when opened.

**Comment by Heckie75** [comment]
@Windyjunk Yes, the website has changed. I've just fixed the addon. The pull request is in review, see https://github.com/xbmc/repo-plugins/pull/4455

If you can't wait or want to test the changes you can download the addon here: https://github.com/Heckie75/kodi-addon-bbc-podcasts/blob/main/plugin.audio.bbcpodcasts.2.0.6.zip
